transform/transformers/InMemoryZip.py

```diff
@@ -0,0 +1,30 @@
+from io import BytesIO
+from zipfile import ZIP_DEFLATED, ZipFile
+
+
+class InMemoryZip(object):
+    """Class for creating in memory Zip objects using BytesIO."""
+    def __init__(self):
+        self.in_memory_zip = BytesIO()
+
+    def append(self, filename_in_zip, file_contents):
+        """Appends a file with name filename_in_zip and contents of
+        file_contents to the in-memory zip."""
+        # Get a handle to the in-memory zip in append mode
+        zf = ZipFile(self.in_memory_zip, "a", ZIP_DEFLATED, False)
+
```

Comment:
What happens if this throws an error?

Reply:
Not sure what to do here . Currently these bubble up and are caught in main where they get logged and returned as a 500 . I have added another exception clause on the Images route so we always catch the errors , but since we cant do anything about them I think handlers on the endpoint might be the correct place to catch them ?